.github/workflows/main.yml

```diff
@@ -90,7 +95,6 @@ jobs:
           CC: ${{ matrix.config.compiler }}
         run: |
           cmake -B build -G Ninja -DCMAKE_BUILD_TYPE=Release \
-                -DCMAKE_INSTALL_PREFIX=/usr \
```

Comment:
@fabiangreffrath It seems with this line CPack couldn't find Homebrew libs.

Reply:
I think this was needed to install/find the soundfont file. 🤔